packages/react-components/src/components/styles/RichTextEditor.styles.ts

```diff
@@ -10,6 +10,10 @@ import { RichTextEditorStyleProps } from '../RichTextEditor/RichTextEditor';
 export const richTextEditorStyle = (props: { minHeight: string; maxHeight: string }): string => {
   return mergeStyles({
     border: 'none',
+    fontSize: '0.875rem',
```

Comment:
Could you please add comment why we need to set fontSize here now?

Reply:
This is for consistency with the non-RTE text box.